docs/how-to-connect-gpt-extensions.md

```diff
@@ -4,151 +4,157 @@ First to integrate this extension, you need to use the following versions of the
 
     @gravity-ui/markdown-editor version 13.18.0 or higher
 
-### 1. Add extension usage to extension builder 
+Featurs:
 
-```ts
-    import {gptExtension} from '@gravity-ui/markdown-editor';
+![Custom prompt preset](./assets/gifs/custom-prompt-preset-gpt.gif)
+![Prompt preset](./assets/gifs/prompt-preset-gpt.gif)
```

Comment:
Limit the width, choose the most advantageous width, I think it should be about 400-500 px

Reply:
Let's add these pictures to the 5th paragraph after `### 4. Done, You can use the extension!` section, add a short text explaining what is happening in the pictures